plugin-test/build.gradle

```diff
@@ -15,14 +15,16 @@ sourceCompatibility = 1.8
 
 buildscript {
     repositories {
-        flatDir dirs: "../jars/repo/me/rexim/morganey/gradle/morganey-gradle-plugin/unspecified/"
-        mavenCentral()
+        maven {
+            url "https://plugins.gradle.org/m2/"
+        }
     }
-
     dependencies {
-        classpath 'me.rexim.morganey.gradle:morganey-gradle-plugin:unspecified'
```

Comment:
I can't see the reason of changing our package name (prepending this `gradle.plugin` there). Could you please elaborate?